src/Core/Json.re

```diff
@@ -18,10 +18,12 @@ module Decode = {
         string => result(value, string)
     );
 
-    let of_file = file =>
+    let of_file = file => {
+      Console.log(file);
```

Comment:
We'll need to remove the debug logging

Reply:
Oops! I thought I had removed all of them. Thanks for pointing this one out! Fixed now 👍